src/ConfiguredMediator.php

```diff
@@ -37,15 +44,43 @@ public function uninstall(Composer $composer, IOInterface $io)
 
     public function installOrUpdateFunction(PackageEvent $event): void
     {
+        $gnuPG = $this->createGnuPG();
+        // we do not want to crash if no GnuPG was found
+        // but display a noticeable warning to the user
+        if (null === $gnuPG && !self::$warningDisplayed) {
+            $this->io->write(
```

Comment:
Shouldn't we write this in more ... **bold**?

Like use `$this->io->write('<warning>' . ... . '</warning>');`?

Reply:
There is no `<warning>` format by default. We could certainly add one if deemed necessary.

Reply:
Hm... According to [BaseIO](https://github.com/composer/composer/blob/378a5b72b9f81e8e919e41ecd3add6893d14b90e/src/Composer/IO/BaseIO.php#L276) there seems to be one...

That was where I got the idea from...

Reply:
I'll give it a try :)

Reply:
Actually you are right there is a `<warning>` set up.  So I changed the warning to `<warning>` but I left the second one at `<comment>` looked a bit to flashy in my eyes.